.github/workflows/build.yml

```diff
@@ -64,7 +64,7 @@ jobs:
       - uses: actions/checkout@v3
       - name: Set up Docker Buildx
         id: buildx
-        uses: docker/setup-buildx-action@c74574e6c82eeedc46366be1b0d287eff9085eb6
+        uses: docker/setup-buildx-action@8c0edbc76e98fa90f69d9a2c020dcb50019dc325
```

Comment:
these don't belong in this PR (and neither does the commit message :-)). `git rebase origin/main` should get rid of these.